tracer/src/Datadog.Trace/Util/UriHelpers.cs

```diff
@@ -162,6 +162,7 @@ public static bool IsIdentifierSegment(string absolutePath, int startIndex, int
                         }
 
                         continue;
+                    case ',':
```

Comment:
I'm not sure if I follow
Are you asking if hex segments: > 16 characters, that contains at least one letter and one comma that we shouldn't obfuscate it?

Reply:
I'm saying that I've never seen a 123af,12a for an hexa value. So if I'm right, then in the case where we find a letter and a comma, then it's not an id. So we should not obfuscate it. 
Also, what about points? Should we add them here? 
(Maybe that's a good time to go and look at what others tracers have done to see if we are close as well.)

Reply:
>Maybe that's a good time to go and look at what others tracers have done to see if we are close as well 

I need to do that as part of the RFC on obfuscating outgoing Http requests too, so would be good to document them all!

Reply:
> I'm saying that I've never seen a 123af,12a for an hexa value. So if I'm right, then in the case where we find a letter and a comma, then it's not an id. So we should not obfuscate it.

Okay that makes sense. 

So `"123af,12a"` currently doesn't get obfuscated because the `segmentLength` is less than 16, but you'd also expect that to still return false if the `segmentLength` is greater than 16 because it is no longer an "id".
Something like `"123afafafafaf,12a"` does get obfuscated with `?` currently, but ideally it wouldn't. Should be able to add a check for that

I will go through the other tracers and see if I can see what they do and will document along the way :)

Reply:
> I need to do that as part of the RFC on obfuscating outgoing Http requests too, so would be good to document them all!

:itsatrap: :p 

> Something like "123afafafafaf,12a" does get obfuscated with ? currently, but ideally it wouldn't. Should be able to add a check for that

I assume yes.

> I will go through the other tracers and see if I can see what they do and will document along the way :)

Thanks a lot. Don't hesitate to ask for pointers in #apm-integrations. Also, you can use  [codesearch](https://cs.github.com/?scopeName=DataDog&scope=org%3ADataDog&q=%3F), but that doesn't prevent you to ask to others if the search isn't obvious